docs/user_guides/fs/feature_group/deprecation.md

```diff
@@ -0,0 +1,74 @@
+---
+description: Documentation on how to deprecate a feature group in Hopsworks and the different APIs available to interact with them.
+---
+
+# How to deprecate a Feature Group
+
+### Introduction
+
+To disparage the usage of specific feature groups it is possible to deprecate them.
+When a feature group is deprecated user will be warned when employing it or feature views created using it.
```

Comment:
`When a feature group is deprecated, user will be warned when they try to use it or use a feature view that depends on it.